cli/tests/environment-activate.bats

```diff
@@ -532,3 +548,29 @@ env_is_activated() {
   run "$FLOX_BIN" activate -- echo '$foo'
   assert_output "bar"
 }
+
+
+# ---------------------------------------------------------------------------- #
+
+@test "bash: 'profile.common' is sourced before 'profile.bash'" {
+  "$FLOX_BIN" delete -f
+  "$FLOX_BIN" init
+  "$FLOX_BIN" edit -f "$BATS_TEST_DIRNAME/activate/profile-order.toml"
+  SHELL="bash" run bash -c '"$FLOX_BIN" activate -- mkdir foo'
```

Comment:
`mkdir foo` is a dummy command that allows you to non-interactively activate the environment. In short, it's just there so that I can get the activation scripts to run. If you give it an empty command after the `--` it just puts you in the interactive shell (at least when run interactively).

Reply:
I would prefer `true`

Reply:
Fixed